pkg/client/group_mgmt_client_test.go

```
@@ -21,7 +22,7 @@ func TestNewClient(t *testing.T) {
 
 	// Create client
 	var err error
-	client, err = NewClient("x.x.x.x", "xxx", "xxx", "v1", true, false)
+	client, err = NewClient(os.Getenv("GO_SDK_TEST_HOST"), os.Getenv("GO_SDK_TEST_USER"), os.Getenv("GO_SDK_TEST_USER_PASSWORD"), "v1", true, false)
```

The test should fail gracefully if the environment variables are not defined.  We could produce a message detailing the environmental requirements.